### DISCUSSION TITLE
Сosmetic improvements

### DISCUSSION THREAD

**Discussion by Enzokot** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#222] [General]
As a suggestion. Remove underscores in the name of the component configuration reload command. Like other commands in this section
![image](https://user-images.githubusercontent.com/43236573/149664924-a2b86f27-30b6-4c0e-aa7a-1381c39e7954.png)

**Comment by PiotrMachowski** [comment]
Yeeeah, I have taken a look at it some time ago, it's because other platforms use their own implementations of reload, but map extractor uses implementation that already exists in HA. Maybe I will adjust it when I will introduce bigger changes in this integration

**Comment by Enzokot** [comment]
Thanks!
